contrib/nitrpg/src/templates/panels.nit

```diff
@@ -98,6 +99,82 @@ class ErrorPanel
 
 end
 
+# A panel that display a markdown content rendered as HTML.
+class MDPanel
+	super Panel
+
+	# Markdown text to display.
+	var text: String
+
+	redef fun rendering do
+		add """<div class="panel">
+			    <div class="panel-body">{{{text.md_to_html}}}</div>
+			  </div>"""
+	end
+end
+
+# Display a list of active game.
+#
+# Used for NitRPG homepage.
+class GamesShortListPanel
+	super Panel
+
+	# Root url used for links.
+	var root_url: String
+
+	# List of NitRPG games to display.
+	var games: Array[Game]
+
+	redef fun render_title do
+		add "<span class=\"glyphicon glyphicon-home\"></span>&nbsp;&nbsp;"
+		add "<a href=\"{root_url}/games\">Active games</a>"
+	end
+
+	redef fun render_body do
+		if games.is_empty then
+			add "<em>No game yet...</em>"
+			return
+		end
+		for game in games do
+			add "{game.link} ({game.load_players.length} players)<br>"
+		end
+	end
+end
+
+# A panel that display a list of player in a repo.
+class GamesListPanel
+	super GamesShortListPanel
+	super TablePanel
+
+	redef fun render_title do
+		add "<span class=\"glyphicon glyphicon-home\"></span>&nbsp;&nbsp;"
+		add "<a href=\"{root_url}/games\">Active games</a>"
+	end
+
+	redef fun render_body do
+		if games.is_empty then
+			add "<div class=\"panel-body\">"
+			add "<em>No player yet...</em>"
+			add "</div>"
+			return
+		end
+		add """<table class="table table-striped table-hover">
+			    <tr>
+				 <th>Game</th>
+				 <th>Players</th>
+				 <th>Achievements</th>
+				</tr>"""
+		for game in games do
+			add "<tr>"
+			add " <td>{game.link}</td>"
+			add " <td>{game.load_players.length}</td>"
+			add " <td>{game.load_achievements.length}</td>"
+			add "</tr>"
```

Comment:
I have trouble to determine when you prefer double-quoted-strings and triple-quoted-strings.